packages/web-api/src/WebClient.ts

```diff
@@ -395,24 +389,25 @@ export class WebClient extends Methods {
     })();
   }
 
-  /* eslint-disable no-trailing-spaces */
   /**
    * This wrapper method provides an easy way to upload files using the following endpoints:
-   * 
+   *
    * **#1**: For each file submitted with this method, submit filenames
    * and file metadata to {@link https://api.slack.com/methods/files.getUploadURLExternal files.getUploadURLExternal} to request a URL to
    * which to send the file data to and an id for the file
-   * 
+   *
    * **#2**: for each returned file `upload_url`, upload corresponding file to
    * URLs returned from step 1 (e.g. https://files.slack.com/upload/v1/...\")
-   * 
+   *
    * **#3**: Complete uploads {@link https://api.slack.com/methods/files.completeUploadExternal files.completeUploadExternal}
-   * 
+   *
    * **#4**: Unless `request_file_info` set to false, call {@link https://api.slack.com/methods/files.info files.info} for
    * each file uploaded and returns that data. Requires that your app have `files:read` scope.
    * @param options
    */
-  public async filesUploadV2(options: FilesUploadV2Arguments): Promise<WebAPICallResult> {
+  public async filesUploadV2(options: FilesUploadV2Arguments): Promise<
+  WebAPICallResult & { files: FilesCompleteUploadExternalResponse[] | WebAPICallResult[] }
```

Comment:
Small change to make this part typesafe as well.

Reply:
good catch